firefox-ios/Client/Frontend/Translations/TranslationsAction.swift

```diff
@@ -0,0 +1,23 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Common
+import Redux
+
+struct TranslationsAction: Action {
+    let windowUUID: WindowUUID
+    let actionType: ActionType
+
+    init(
+        windowUUID: WindowUUID,
+        actionType: any ActionType,
+    ) {
+        self.windowUUID = windowUUID
+        self.actionType = actionType
+    }
+}
+
+enum TranslationsActionType: ActionType {
+    case didTapOnRetry
```

Comment:
```suggestion
    case didTapRetryFailedTranslation
```

Since this is a bit separate from the other types, a bit of extra context might be useful... 🤔

Reply:
Updated here 99816dbe35973c352b03089ed716b2abe82ebcdb